browser/modulevu.go

```diff
@@ -16,6 +17,7 @@ type moduleVU struct {
 	k6modules.VU
 
 	*pidRegistry
+	remoteRegistry *registry.RemoteRegistry
```

Comment:
What do you think about just using composition as we do with `pidRegistry`? E.g.:
```suggestion
	*registry.RemoteRegistry
```
I like the idea of making the `moduleVU` group methods that belong to that specific VU/iteration execution and being able to call them directly. E.g.: `vu.IsRemoteBrowser()` instead of `vu.remoteRegistry.IsRemoteBrowser()`. But I understand this is pretty personal opinion.